tools/model_tools/src/openvino/model_zoo/_common.py

```diff
@@ -135,25 +135,30 @@ def get_package_path(python_executable, package_name):
     return file_path.parent, completed_process.stderr
 
 def get_version():
-    if VERSION_FILE and VERSION_FILE.is_file():
-        with VERSION_FILE.open('r') as version_file:
-            version = version_file.readline().rstrip('\n')
-        return f'commit {version}'
-    else:
-        return 'unknown'
+    try:
+        from openvino.runtime import get_version
+        version = get_version()
+        version_match = re.match(r"^([0-9]+).([0-9]+)*", version)
```

Comment:
it maybe not be very robust and informative if you provide ov version instead of your own (e.g. using development state of the tool from Github with already released ov package), can we combine ov version and your as common solution?

Reply:
Yes, thanks, updated version